src/Foundation/Grid.elm

```diff
@@ -1,29 +1,39 @@
-module Foundation.Grid
-    exposing
-        ( container
+module Foundation.Grid exposing ( container
         , containerFluid
         , containerFull
-        , gridx
-        )
+        , grid
+        , cell
+    )
+import Foundation.Classnames exposing (..)
+import String exposing (..)
 
 import Html exposing (Html, div, Attribute)
 import Html.Attributes exposing (class, classList)
 
 
 container : List (Attribute msg) -> List (Html msg) -> Html msg
 container attributes children =
-    div ([ class "grid-container" ] ++ attributes) children
+    div ([ class gridContainer ] ++ attributes) children
 
 containerFluid : List (Attribute msg) -> List (Html msg) -> Html msg
 containerFluid attributes children =
-    div ([ class "grid-container fluid" ] ++ attributes) children
+    div ([ class gridContainerFluid ] ++ attributes) children
 
 
 containerFull : List (Attribute msg) -> List (Html msg) -> Html msg
 containerFull attributes children =
-    div ([ class "grid-container full" ] ++ attributes) children
+    div ([ class gridContainerFull ] ++ attributes) children
 
 
-gridx : List (Attribute msg) -> List (Html msg) -> Html msg
-gridx attributes children =
-    div ([ class "grid-x" ] ++ attributes) children
+grid : List (String) -> List (Html msg) -> Html msg
```

Comment:
I don't like that `grid` can accept any string. For classes it is fine, but grid type should be a distinct type.